### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by LinuxTux193** [RoganDawes/P4wnP1_aloa#318]
The last commit is from February 2020 and many issues are not resolved, is this project dead?

**Comment by Sabering1** [comment]
yes

**Comment by nototter** [comment]
yes; but theres an alternative (not the best though): https://github.com/RoganDawes/P4wnP1_aloa/issues/320
